### PR TITLE
Added Perun documentation section

### PR DESCRIPTION
**Pull request by zlamalp**
# Summary

This PR adds page about VO management in Perun in the context of EGI Check-In service. So far it was completely missing.

Right now it is a draft meant for further discussions about the content structure, placement etc.

---
**Related issue :** none

### CODE REVIEW

**Comment by gwarf** [comment]
Thanks @zlamalp !
As this is your first contribution to this repo I will have to approve the workflow run, once they have run please address the issues reported, let me know if you need some help or find something confusing.
You can look at the contributing guide pages: https://docs.egi.eu/about/contributing/
@vardizzo-lab could you please review the integration of this documentation?

**Comment by zlamalp** [comment]
Hi @gwarf, can you please approve the workflow run again? I believe I've fixed lint errors. I'd like to get new version deployed in the preview. Thank you.

**Comment by gwarf** [comment]
Sure @zlamalp , just did it, thanks for the reminder.
I also update the PR with the latest changes from the main branch.

**Comment by gwarf** [comment]
OK, so now all is green, thanks @zlamalp !
Please let us know when this PR will be ready for review.

**Comment by andrea-manzi** [comment]
@zlamalp  i see that some links in HOW-TO sections are pointing to sections in your docs still to be completed. Maybe better to hide them for the moment?

**Comment by zlamalp** [comment]
@andrea-manzi HOW-TOs, which are not ready yet are now hidden, I will re-enable them once they are ready.

**Comment by andrea-manzi** [comment]
> @andrea-manzi HOW-TOs, which are not ready yet are now hidden, I will re-enable them once they are ready.

@zlamalp thanks a lot, let's wait for Valeria's review and then approve the PR

**Comment by gwarf** [comment]
If we want to do things cleanly we may want to split the Check-in/COmanage specific content from https://docs.egi.eu/documentation/398/users/aai/check-in/vos/ to a sub page similar to the one proposed here for PERUN, and then add some introductory text in https://docs.egi.eu/documentation/398/users/aai/check-in/vos/ describing the various possibilities (PERUN, Check-in/COmanage and VOMS)

**Comment by gwarf** [comment]
@vardizzo-lab can you please have a look?

**Comment by andrea-manzi** [comment]
@vardizzo-lab what is missing to include this doc?

**Review by vardizzo-lab — CHANGES_REQUESTED** [review]

File: content/en/users/aai/check-in/vos/perun/_index.md
Comment (zlamalp) [reply content/en/users/aai/check-in/vos/perun/_index.md]: Fixed in latest version.

File: content/en/users/aai/check-in/vos/perun/_index.md
Comment (zlamalp) [reply content/en/users/aai/check-in/vos/perun/_index.md]: Fixed in latest version.

File: content/en/users/aai/check-in/vos/perun/_index.md
Comment (zlamalp) [reply content/en/users/aai/check-in/vos/perun/_index.md]: Features not used in EGI are removed in latest version.

File: content/en/users/aai/check-in/vos/perun/_index.md
Comment (zlamalp) [reply content/en/users/aai/check-in/vos/perun/_index.md]: Features not used in EGI are removed in latest version.

File: content/en/users/aai/check-in/vos/perun/_index.md
Comment (zlamalp) [reply content/en/users/aai/check-in/vos/perun/_index.md]: Part about personal certificates moved to FAQ.

**Comment by zlamalp** [comment]
Can someone please approve the build/check workflow so it gets deployed for the preview? Thank you.

**Comment by gwarf** [comment]
Sure it's done, I overlooked this.
Once this PR will have been merged, the following ones will run the workflow automatically. :)

**Review by vardizzo-lab — APPROVED** [review]

**Pull request merged**
